composer.json

```diff
@@ -1,8 +1,14 @@
 {
+    "config": {
+        "platform": {
+            "php": "5.6.29"
```

Comment:
5.6 should be enough / better - will doublecheck

Reply:
Yes, we are up to about ``5.6.33`` now, and counting for another 6 months.